machAeroTutorials/airfoilopt_ffd.rst

```diff
@@ -51,7 +51,10 @@ The FFD box can now be set up.
     :end-before: # rst FFDBox2
 
 
-``nffd`` signifies the number of chordwise points. An empty FFD box is created.
+``nffd`` signifies the number of chordwise slices.
+The `2`'s in `(nffd, 2, 2, 3)` are the number of slices in the "spanwise" and "thickness" directions.
```

Comment:
I would rephrase like: "we pre-allocate an array of generic size (a,b,c,3) to set up an empty FFD box. In this example, a=nffd (number of chordwise sections), b=c=2 (number of spanwise and thickness-wise sections resepctively) and the final 3 is "fixed" as we are using 3D coordinates for each point"

not sure if this is clearer/nicer, up to you!

Reply:
I changed it to match what you suggested